local_groups/views.py

```diff
@@ -22,46 +29,16 @@ def get_context_data(self, **kwargs):
 
 
 # View for Admin updates to Group Info
-class GroupUpdateView(LoginRequiredMixin, UpdateView):
+@method_decorator(verified_email_required, name='dispatch')
+class GroupManageView(LoginRequiredMixin, SuccessMessageMixin, UpdateView):
     model = Group
-    fields = [
-        'description',
-        'city',
-        'state',
-        'county',
-        'country',
-        'postal_code',
-        'size',
-        # Edit group email instead of rep email so BSD auth still works
-        'group_contact_email',
-        'rep_first_name',
-        'rep_last_name',
-        'rep_phone',
-        'rep_postal_code',
-        'website_url',
-        'facebook_url',
-        'twitter_url',
-        'instagram_url',
-        'other_social',
-        'types_of_organizing',
-        'other_types_of_organizing',
-        'issues',
-        'other_issues',
-        'constituency',
-        'last_meeting',
-        'recurring_meeting',
-        'meeting_address_line1',
-        'meeting_address_line2',
-        'meeting_city',
-        'meeting_state_province',
-        'meeting_postal_code',
-        'meeting_country',
-    ]
-    template_name_suffix = '_update_form'
+    form_class = GroupManageForm
+    success_message = "Your group has been updated successfully."
+    template_name_suffix = '_manage_form'
 
     # Redirect to same page on success
     def get_success_url(self):
-        return "/groups/" + self.object.slug + "/update/"
+        return reverse_lazy('groups-manage', kwargs={'slug': self.object.slug})
```

Comment:
Might need to clean up merge with https://github.com/Our-Revolution/site/pull/72/files#diff-7984a20a9d14ef5f869b494af09997c1R64